Added vial support for For_Science

arko9699 opened the pull request.

arko9699 committed:
Added vial support for For_Science

arko9699 committed:
Update rules.mk

arko9699 committed:
Merge branch 'vial-kb:vial' into vial

arko9699 committed:
Disabled QMK features to keep firmware size in check for AVR

xyzz commented on keyboards/for_science/keyboard.json:
```diff
@@ -14,8 +14,8 @@
     },
     "features": {
         "bootmagic": true,
-        "command": true,
```
do not modify files outside of the vial keymap that also exist in upstream qmk, this will cause conflicts during merge

arko9699 committed:
moved qmk feature flags from keyboard.json to vial/rules.mk

Moved QMK feature disables to vial/rules.mk to maintain compatibility with upstream QMK.

xyzz commented:
there are still modifications outside of the vial keymap

arko9699 committed:
Delete rules.mk

arko9699 committed:
Merge branch 'vial-kb:vial' into vial

arko9699 commented:
I'm sorry for the frankly egregious series of errors I've made. It should hopefully be correct now.
